src/pybind/mgr/dashboard_v2/frontend/src/app/shared/cd-password-button.directive.ts

```diff
@@ -0,0 +1,42 @@
+import { Directive, ElementRef, Input, HostListener, Renderer2 } from '@angular/core';
+
+@Directive({
+  selector: '[cd-password-button]'
+})
+export class CdPasswordButtonDirective {
```

Comment:
Class name should not contain the application prefix.

Reply:
Fixed